src/components/Waveform/WaveformTypes.ts

```diff
@@ -51,4 +51,6 @@ export interface IWaveformRef {
   stopRecord: () => Promise<string>;
   pauseRecord: () => Promise<boolean>;
   resumeRecord: () => Promise<boolean>;
+  currentState: PlayerState | RecorderState;
+  playerKey: () => string;
```

Comment:
```suggestion
  playerKey: string;
```
In tandem with what I have commented in `Waveform.tsx` file.